src/tests/Microsoft.PowerFx.Connectors.Tests.Shared/Responses/ZD Tickets GetSchema.json

```diff
@@ -18,254 +18,308 @@
-          "type": "string",
           "x-ms-permission": "read-write",
           "x-ms-sort": "none"
         },
         "type": {
+          "type": "string",
+          "x-ms-enum-values": [
+            {
+              "value": "problem",
+              "displayName": "Problem"
+            },
+            {
+              "value": "incident",
+              "displayName": "Incident"
+            },
+            {
+              "value": "question",
+              "displayName": "Question"
+            },
+            {
+              "value": "task",
+              "displayName": "Task"
+            }
+          ],
           "title": "type",
           "x-ms-capabilities": { "filterFunctions": [ "lt", "le", "eq", "ne", "gt", "ge", "min", "max", "countdistinct", "contains", "startswith", "endswith", "length", "indexof", "replace", "substring", "substringof", "tolower", "toupper", "trim", "concat" ] },
-          "type": "string",
           "x-ms-permission": "read-write",
           "x-ms-sort": "none"
         },
         "subject": {
+          "type": "string",
           "title": "subject",
           "x-ms-capabilities": { "filterFunctions": [ "lt", "le", "eq", "ne", "gt", "ge", "min", "max", "countdistinct", "contains", "startswith", "endswith", "length", "indexof", "replace", "substring", "substringof", "tolower", "toupper", "trim", "concat" ] },
-          "type": "string",
           "x-ms-permission": "read-write",
           "x-ms-sort": "none"
         },
         "raw_subject": {
+          "type": "string",
           "title": "raw_subject",
           "x-ms-capabilities": { "filterFunctions": [ "lt", "le", "eq", "ne", "gt", "ge", "min", "max", "countdistinct", "contains", "startswith", "endswith", "length", "indexof", "replace", "substring", "substringof", "tolower", "toupper", "trim", "concat" ] },
-          "type": "string",
           "x-ms-permission": "read-write",
           "x-ms-sort": "none"
         },
         "description": {
+          "type": "string",
           "title": "description",
           "x-ms-capabilities": { "filterFunctions": [ "lt", "le", "eq", "ne", "gt", "ge", "min", "max", "countdistinct", "contains", "startswith", "endswith", "length", "indexof", "replace", "substring", "substringof", "tolower", "toupper", "trim", "concat" ] },
-          "type": "string",
           "x-ms-permission": "read-write",
           "x-ms-sort": "none"
         },
         "priority": {
-          "title": "priority",
-          "x-ms-capabilities": { "filterFunctions": [ "lt", "le", "eq", "ne", "gt", "ge", "min", "max", "countdistinct", "contains", "startswith", "endswith", "length", "indexof", "replace", "substring", "substringof", "tolower", "toupper", "trim", "concat" ] },
           "type": "string",
-          "enum": [
-            "low",
-            "normal",
-            "high",
-            "urgent"
+          "x-ms-enum-values": [
```

Comment:
>"x-ms-enum-values": [](http://example.com/codeflow?start=10&length=19)

Do we need a test case for x-ms-enum? 

Are we losing coveraging dropping "enum" on line 95?

Reply:
already tested with action connectors (this code is shared)
dropping "enum" is not an issue as this extension has precedence (that's how i define it - I can't prove it)